index.html

```diff
@@ -1,431 +1,247 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-CD9JE7NBNV"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+      function gtag(){window.dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'G-CD9JE7NBNV');
+    </script>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="stylesheet"
-      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
-      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
-      crossorigin="anonymous"
-    />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
   <body>
-    <main>
-      <header id="header">
-        <div class="header-container">
-          <div class="logo">
-            <a class="rotate-logo" href="#">My Logo</a>
-          </div>
-          <nav class="navbar">
-            <ul class="d-navbar">
-              <li><a class="a-navbar" href="#Portfolio">Portfolio</a></li>
-              <li><a class="a-navbar" href="#About">About</a></li>
-              <li><a class="a-navbar" href="#Contact">Contact</a></li>
-            </ul>
-            <div class="humberger">
-              <i class="fa fa-bars"></i>
+  <!--Mobile menu-->
+  <div id="menuId" class="myMenu">
+    <ul >
+      <li class="claList"><a href="#portfolio">Portfolio</a></li>
+      <li class="claList"><a href="#about">About</a></li>
+      <li class="claList"><a href="#contact">Contact</a></li>
+    </ul>
+    <img src="./image/IconX.svg" alt="Close Menu" class="close" id="closeMenu">
+  </div>
+  <!--Mobile menu End-->
+  <div class="card-popup-window">
```

Comment:
Technologies in the pop-up section have a little bit more padding which does not match with the design. Moreover, icons do are not aligned vertically with text inside buttons. Kindly see the screenshot below and fix the design issues.
![pop-up-btns](https://user-images.githubusercontent.com/35504149/132063665-94537b76-c3e7-423a-b3fa-56c5f1b2b022.png)